doc/fiware-tour-guide-application-a-tutorial-on-how-to-integrate-the-main-fiware-ges/managing-iot-data.md

```diff
@@ -1 +1,558 @@
+                	"object_id": "t",
+                	"name": "temperature:kitchen",
+                	"type": "Number"
+            	}
+        	]
+    	}
+	]
+}
+```
+`ENTITY_ID` will be the Id of the restaurant to which we want to add the sensor, and `DEV_ID` will be the Id of the sensor.  In this example, we'll use the restaurant with Id `0115206c51f60b48b77e4c937835795c33bb953f`.  The Id of the sensor will be a compound of the restaurant Id, the room and the type of the sensor, like `0115206c51f60b48b77e4c937835795c33bb953f-kitchen-temperature`.
+
+As with the service, we need to add the following HTTP headers in our request:
+
+- Fiware-Service, that for TourGuide-App will have a value of `tourguide`,
+- Fiware-ServicePath, that will be the organization of the restaurant, in our example `/Franchise1`,
+- Content-type of the data we are POSTing, this will be `application/json`.
+
+We do the request:
+```
+$ (curl -v -H 'content-type: application/json' -H 'fiware-service: tourguide' -H 'fiware-servicepath: /Franchise1' -X POST 'http://localhost:4041/iot/devices' -d @- ) << EOF
+{
+	"devices": [
+    	{
+        	"device_id": "0115206c51f60b48b77e4c937835795c33bb953f-kitchen-temperature",
+        	"entity_name": "0115206c51f60b48b77e4c937835795c33bb953f",
+        	"protocol": "UL20",
+        	"entity_type": "Restaurant",
+        	"timezone": "Europe/Madrid",
+        	"attributes": [
+            	{
+                	"object_id": "t",
+                	"name": "temperature:kitchen",
+                	"type": "Number"
+            	}
+        	]
+    	}
+	]
+}
+EOF
+```
+and the response should be `{}` if there was no error.  The log of the IoT Agent should have entries like this:
+```
+time=2016-09-29T12:55:54.992Z | lvl=DEBUG | corr=3a59f5cc-3e0f-4d4b-9249-04145eeb3fe3 | trans=3a59f5cc-3e0f-4d4b-9249-04145eeb3fe3 | op=IoTAgentNGSI.RestUtils | srv=tourguide | subsrv=/Franchise1 | msg=Request for path [/iot/devices] from [localhost:4041] | comp=IoTAgent
+time=2016-09-29T12:55:54.992Z | lvl=DEBUG | corr=3a59f5cc-3e0f-4d4b-9249-04145eeb3fe3 | trans=3a59f5cc-3e0f-4d4b-9249-04145eeb3fe3 | op=IoTAgentNGSI.GenericMiddlewares | srv=tourguide | subsrv=/Franchise1 | msg=Body:
+
+{
+	"devices": [
+    	{
+        	"device_id": "0115206c51f60b48b77e4c937835795c33bb953f-kitchen-temperature",
+        	"entity_name": "0115206c51f60b48b77e4c937835795c33bb953f",
+        	"protocol": "UL20",
+        	"entity_type": "Restaurant",
+        	"timezone": "Europe/Madrid",
+        	"attributes": [
+            	{
+                	"object_id": "t",
+                	"name": "temperature:kitchen",
+                	"type": "Number"
+            	}
+        	]
+    	}
+	]
+}
+
+ | comp=IoTAgent
+time=2016-09-29T12:55:54.993Z | lvl=DEBUG | corr=3a59f5cc-3e0f-4d4b-9249-04145eeb3fe3 | trans=3a59f5cc-3e0f-4d4b-9249-04145eeb3fe3 | op=IoTAgentNGSI.RestUtils | srv=tourguide | subsrv=/Franchise1 | msg=Handling device provisioning request. | comp=IoTAgent
```

Comment:
can we make this shorter by only showing the key log trace or the couple of ... ?